kratos/includes/fixed_size_memory_pool.h

```diff
@@ -64,8 +64,9 @@ namespace Kratos
 		  : LockObject()
 		  , mChunkSize(ChunkSize)
 	  {
-		  for (int i_thread = 0; i_thread < OpenMPUtils::GetCurrentNumberOfThreads(); i_thread++)
-			  mThreadsPool.emplace_back(BlockSizeInBytes, ChunkSize, i_thread);
+		  for (int i_thread = 0; i_thread < OpenMPUtils::GetCurrentNumberOfThreads(); i_thread++) {
+			//   mThreadsPool.push_back(BlockSizeInBytes, ChunkSize, i_thread);
```

Comment:
@pooyan-dadvand without copy and move ctor this basically cannot be done, I think this needs to be refactored somehow